### ISSUE TITLE
integer key max value

### ISSUE THREAD

**Issue by javierfjc** [hivedb/hive#1204]
Hi,
Why exists a max limit to integer key value of 0xFFFFFFFF ?

I have greaters keys values that I can´t use. With string keys is allowed lenght until 255 characters

thanks

**Comment by Quantam-Studios** [comment]
"The supported integer values include all integers between -2^53 and 2^53, and some integers with larger magnitude" - https://docs.hivedb.dev/#/more/limitations is all I can find on this. It probably has something to do with the limits of integer and double types across all programming languages. 
A good explanation I found is in the Safe Integers section of this page: https://exploringjs.com/es5/ch11.html

**Comment by Quantam-Studios** [comment]
Is it not between -0xFFFFFFFF - 0xFFFFFFFF

**Comment by Quantam-Studios** [comment]
Yes you are correct, and it has a reason beyond the creators of Hive's control.
